distbench_engine.cc

```diff
@@ -999,15 +1076,23 @@ void DistBenchEngine::RunAction(ActionState* action_state) {
         [this](std::shared_ptr<ActionIterationState> iteration_state) {
           RunRpcActionIteration(iteration_state);
         };
-  } else if (action.proto.has_activity_name()) {
-    // TODO:
-    // Currently we have only one activity. In future,
-    // make a map of activity_name to function.
+  } else if (action.proto.has_activity_config_name()) {
+    auto* sac =
+        &stored_activity_config_
+            [activity_config_indices_map_[action.proto.activity_config_name()]];
```

Comment:
We could just store the index as part of parsing it, like we do for action.rpc_service_index.